src/GitLink/Program.cs

```diff
@@ -8,6 +8,9 @@
 namespace GitLink
 {
     using System;
+    using System.CommandLine;
```

Comment:
Ya. But it works. I think it's only a prerelease because the .NET team hasn't had time to review it for final publication.